test/unit/traversing.js

```diff
@@ -743,6 +743,37 @@ QUnit.test( "contents()", function( assert ) {
 	assert.equal( c[ 0 ].nodeValue, "hi", "Check node,textnode,comment contents is just the one from span" );
 } );
 
+QUnit.test( "contents() for <template />", function( assert ) {
```

Comment:
In the tests in the `onload` for `<img>` and in the `script` tag just assign something to globals and test that the assignments didn't happen. You need to register the globals you set via `Globals.register` to get them cleaned up automatically at the end of the test, [here is an example test that uses this pattern](https://github.com/jquery/jquery/blob/522f546/test/unit/manipulation.js#L2733-L2757).